apps/wallet/src/ui/app/hooks/useDeepBook.ts

```diff
@@ -0,0 +1,174 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { getActiveNetworkSuiClient } from '_shared/sui-client';
+import { DeepBookClient } from '@mysten/deepbook';
+import { SUI_TYPE_ARG } from '@mysten/sui.js/utils';
+import { useQuery } from '@tanstack/react-query';
+import BigNumber from 'bignumber.js';
+import { useMemo } from 'react';
+
+const FLOAT_SCALING_FACTOR = 1_000_000_000n;
+export const DEFAULT_TICK_SIZE = 1n * FLOAT_SCALING_FACTOR;
+const DEEPBOOK_KEY = 'deepbook';
+
+export const mainnetPools = {
```

Comment:
I'd probably explore a data structure that captures this + coins + coinsmap together so that we could chuck it into growthbook and update it dynamically.